config/prow/cluster/400-crier.yaml

```diff
@@ -32,6 +32,9 @@ spec:
         image: gcr.io/k8s-prow/crier:v20200219-5ea024fc8
         args:
         - --pubsub-workers=5 # Arbitrary number of multiplier
+        - --gcs-workers=1
+        - --gcs-credentials-file=/etc/service-account/service-account.json # TODO(chizhg): use workload identity
+        - --kubernetes-gcs-workers=1
```

Comment:
what's the difference from `gcs-workers`?

Reply:
According to the code, `kubernetes-gcs-workers` is used in reporting `prowjob.json`,  `gcs-workers` is used in reporting `podinfo.json`, and `prowjob.json` is used in spyglass UI if `podinfo.json` does not exist.